Script.js

```diff
@@ -1 +1,6 @@
-document.write("<h2>puto el que lee<h2>");
\ No newline at end of file
+var num1 = window.prompt("Ingrese numero");
+var num2 = window.prompt("Ingrese otro numero");
+
+num1 = parseInt(num1);
+num2 = parseInt(num2);
```

Comment:
This can be abbreviated to 

`const num1 = parseInt(window.prompt("Ingrese numero"))`
`const num2 = parseInt(window.prompt("Ingrese otro numero"))`